src/Database/LSMTree/Internal/Run.hs

```diff
@@ -45,6 +45,7 @@ module Database.LSMTree.Internal.Run (
   , removeReference
   , readBlob
     -- ** Run creation
+  , setNoCache
```

Comment:
Do we want to have this as a separate API or just add a caching argument to the functions that open/create a new run? That way we can't miss any decision.